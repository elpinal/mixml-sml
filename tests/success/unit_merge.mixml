link {unit U : {type a : Type} export (a)} with
{unit U = link {type a : Type} seals {type a = int}}
