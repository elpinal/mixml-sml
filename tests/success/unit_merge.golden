{U : [exist (!7 : Type). {a : [= !7]}]+}
